app/src/androidTest/java/ch/epfl/sdp/blindly/main_screen/my_matches/MyMatchesFragmentTest.kt

```diff
@@ -64,14 +66,14 @@ class WeatherPageTest {
 
     @Test
     fun mapButtonFiresMapActivty() {
-        onView(withId(R.id.open_weather)).check(
+        Espresso.onView(ViewMatchers.withId(R.id.buttonWeatherEvent)).check(
```

Comment:
Wouldn't it be cleaner to import onView to get rid of the Espresso. like it's done in other classes?